dask/array/core.py

```diff
@@ -84,6 +84,15 @@
 )
 from dask.widgets import get_template
 
+try:
+    from dask import threaded
+except (ImportError, ModuleNotFoundError):
```

Comment:
```suggestion
except ImportError:
```

There is no need to catch ``ModuleNotFoundError`` explicitly. It is a subclass of ``ImportError``.

```
>>> ModuleNotFoundError.__mro__
(<class 'ModuleNotFoundError'>, <class 'ImportError'>, <class 'Exception'>, <class 'BaseException'>, <class 'object'>)
```